### PR TITLE
Add JUnit for task commands

### LINKED ISSUE #30: Add JUnit for task commands

### PR DESCRIPTION
**Pull request by aliciatay-zls**
Resolves #30

### CODE REVIEW

File: src/test/java/seedu/duke/commands/MarkAsUndoneCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                + "false" + NEWLINE
+                + "false" + NEWLINE
+                + "true" + NEWLINE
+                + "false" + NEWLINE;
+
+        // checks if tasks were correctly unmarked in task list
+        assertEquals(expectedDone, actualDone);
+
+        System.setIn(originalIn);
+        System.setOut(originalOut);
```
Comment (isaharon) [inline src/test/java/seedu/duke/commands/MarkAsUndoneCommandTest.java:77]: Duplicate statements with the one after?

**Review by isaharon — CHANGES_REQUESTED** [review]
Looks good but do make sure that all testcases passes.

File: src/test/java/seedu/duke/commands/MarkAsUndoneCommandTest.java (hunk 1 above)
Comment (aliciatay-zls) [reply src/test/java/seedu/duke/commands/MarkAsUndoneCommandTest.java:73]: Thank you, edited!

**Review by isaharon — APPROVED** [review]
LGTM!

**Pull request merged**